src/modules/mailer/domain/factories/email.factory.ts

```diff
@@ -0,0 +1,32 @@
+import { Email } from '../entities/email.entity';
+
+export class EmailFactory {
+  static create(data: {
+    to: string[];
+    subject?: string;
+    text?: string;
+    html?: string;
+  }) {
```

Comment:
I have just noticed that the parameter is passed as an object. We also have an agreement that if there are more than 3 parameters, we wrap then in an input object. The same rule was suggested by Robert Martin in his book Clean Code, and we found it quite useful in our work :)